controller/templates/statefulset.yaml

```diff
@@ -104,30 +96,31 @@ spec:
           # case the entrypoint contains a lot of code which has to be executed before the
           # Jupyter server can even accept connections. However, really long running tasks
           # should be put into an init container.
-          # livenessProbe:
-          #   exec:
-          #     command:  ["sh", "-c", "jupyter notebook list | grep :8888/"]
-          #   periodSeconds: 10
-          #   failureThreshold: 6
-          #   timeoutSeconds: 9
-          # readinessProbe:
-          #   exec:
-          #     command: ["sh", "-c", "jupyter notebook list | grep :8888/"]
-          #   periodSeconds: 10
-          #   failureThreshold: 2
-          #   timeoutSeconds: 9
-          # startupProbe:
-          #   exec:
-          #     command: ["sh", "-c", "jupyter notebook list | grep :8888/"]
-          #   periodSeconds: 10
-          #   failureThreshold: 30
-          #   timeoutSeconds: 9
-
-
-        {% if oidc["enabled"] %}
-
+          livenessProbe:
+            httpGet:
+              path: {{ path }}
+              port: 8888
+            periodSeconds: 10
+            failureThreshold: 6
+            timeoutSeconds: 9
+          readinessProbe:
+            httpGet:
+              path: {{ path }}
+              port: 8888
+            periodSeconds: 10
+            failureThreshold: 2
+            timeoutSeconds: 9
+          startupProbe:
+            httpGet:
+              path: {{ path }}
+              port: 8888
+            periodSeconds: 10
+            failureThreshold: 30
+            timeoutSeconds: 9
+        
         - name: oauth2-proxy
           image: "bitnami/oauth2-proxy:7.1.3"
```

Comment:
I know this is not the related to this PR, but I'm noticing now: the official Docker image of oauth2-proxy is stored in quay.io: https://quay.io/oauth2-proxy/oauth2-proxy maybe we should consider using that image? As it would give us some more guarantee of the contents?

Reply:
I like about the Bitnami images that they usually do not run as root. Other than that I can not think of any reason to not use the "official" image. Feel free to open an issue for it.